x-pack/plugins/uptime/common/runtime_types/monitor/state.ts

```diff
@@ -76,9 +17,31 @@ export const StateType = t.intersection([
       port: t.number,
       scheme: t.string,
     }),
+    summaryPings: t.array(PingType),
```

Comment:
PingType contains way too many fields, i think we should do _source when getting ping type for summaries, 
![image](https://user-images.githubusercontent.com/3505601/86739267-0b629b00-c036-11ea-853b-afaa399d8998.png)

Out of all these fields, i only see @timestamp, monitor, observer, summary , tags and urls being used in UI, request size, simple request can become as high as 30kb . i think we should remove unused fields.

Reply:
I hear your point, but that size isn't particularly worrying. By contrast, the elastic.co homepage contains 116k of markup alone. If you're worried about performance have you quantified the impact? The fields are essentially free to add given the queries we're doing, so I'm not worried on the backend.

Adding more types adds complexity in many ways. Additionally, this may one day become a public facing API. Will we want to return so many slightly different types?